app/controllers/form_templates_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class FormTemplatesController < ApplicationController
+  requires_login
+  before_action :ensure_form_templates_enabled
+
+  def index
+    form_templates = FormTemplate.all
+    render_serialized(form_templates, FormTemplateSerializer, root: "form_templates")
+  end
+
+  def show
+    params.require(:id)
+
+    begin
+      template = FormTemplate.find(params[:id])
+    rescue StandardError
+      raise Discourse::NotFound
+    end
```

Comment:
```suggestion
    template = FormTemplate.find_by(id: params[:id])
    raise Discourse::NotFound if !template
```

If we want to avoid using a `begin/rescue` statement, we can use `.find_by`. It's the same as `find`, except it doesn't raise an exception when the record isn't found.